gocd/bci_repo_publish.py

```diff
@@ -131,35 +134,28 @@ def run(self, version, token=None):
             return
 
         # Check openQA results
-        openqa_passed = True
         for pkg in packages:
-            passed = 0
-            pending = 0
-            failed = 0
             for job in self.openqa_jobs_for_product(arch=pkg['arch'], version=version,
                                                     build=pkg['built_version']):
-                if job['result'] in ('passed', 'softfailed'):
-                    passed += 1
-                elif job['result'] == 'none':
+                if job['result'] == 'none':
                     self.logger.info(f'https://openqa.suse.de/tests/{job["id"]} pending')
-                    pending += 1
-                else:
+                    break
+                if job['result'] not in ('passed', 'softfailed'):
                     self.logger.warning(f'https://openqa.suse.de/tests/{job["id"]} failed')
-                    failed += 1
+                    break
+            else:
```

Comment:
"No jobs" must not be treated as "passed", otherwise this would publish before openQA scheduled jobs

Reply:
ok, then we need a different approach on handling the missing s390x and ppc64le tests.